<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>950</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="begin">
     <widget class="QPushButton" name="beginButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>321</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/starttext.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="beginlable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>951</width>
        <height>691</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/begin.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <zorder>beginlable</zorder>
     <zorder>beginButton</zorder>
    </widget>
    <widget class="QWidget" name="break1">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="breaklable">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>-2</y>
        <width>961</width>
        <height>701</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/picture/breakgif.gif</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="moneylable2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>270</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="moneylable1">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>69</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="game">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="quitButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>111</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/quit2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="timelable">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="levellable">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>50</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="mymoneylable2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="aimmoneylable2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="mymoneylable1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="aimmoneylable1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>58</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="bomblabel1">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>70</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="bomblabel2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="bomblabel3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>70</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="bomblabel4">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>70</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="bomblabel5">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>70</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pass">
     <widget class="QLabel" name="passlable">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>-2</y>
        <width>951</width>
        <height>701</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="shop">
     <widget class="QLabel" name="shopbacklable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>951</width>
        <height>691</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/shop1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="shopmanlable">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>260</y>
        <width>261</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/shopmanspeakgif.gif);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="shoptextlable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>691</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/shoptext.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="nextButton">
      <property name="geometry">
       <rect>
        <x>732</x>
        <y>107</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:/picture/nextbutton1.png)
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="fireButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>410</y>
        <width>91</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/firecracker1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="strongButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>400</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/stronger1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="luckButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>410</y>
        <width>111</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/luckier1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="stoneButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>410</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/betterstone1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="diamondButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>410</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/betterdiamond1.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="goodstextlabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>550</y>
        <width>891</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/goodstext.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="firelabel1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>520</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="firelabel2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>520</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="stronglabel1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>520</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="stronglabel2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>520</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lucklabel1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>520</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lucklabel2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>520</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="stonelabel1">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>520</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="stonelabel2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>520</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="diomondlabel1">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>520</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="diomondlabel2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>520</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="shopend">
     <widget class="QLabel" name="shopendlable">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>-2</y>
        <width>951</width>
        <height>701</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="fail1">
     <widget class="QLabel" name="faillable1">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>-2</y>
        <width>951</width>
        <height>691</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="fail2">
     <widget class="QLabel" name="faillabel2">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>-2</y>
        <width>951</width>
        <height>701</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/picture/end.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="replayButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>590</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/endbutton.png);</string>
      </property>
      <property name="text">
       <string>重玩</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>590</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/picture/endbutton.png);</string>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
